### ISSUE TITLE
Document how to integrate shen-elisp/repl with shen-mode

### ISSUE THREAD

**Issue by markokocic** [deech/shen-elisp#15]
I was not able to figure out how to use shen-elisp together with shen-mode in order to be able to load whole file in repl, or to evaluate latest definition in repl.

For SBCL, and other shens it's enough to set `inferior-shen-program`. How to do it for shen-elisp, if possible?

**Comment by deech** [comment]
Yeah I'm not surprised they don't play nice. `shen-mode` was written by another author years before `shen-elisp` came along. It needs a complete rewrite.

**Comment by markokocic** [comment]
I like `shen-mode` because it gives autocomplete, syntax highlight and ability to send the code to be evaluated to the running shen process. This worklfow feels more lispy/emacsy than having to load images from the disk. In ideal world, shen-elisp buffer would behave just like elisp-buffer, and repl might or might not be even needed, something like `ielm` on top.
